src/components/journal-header/journal-header.scss

```diff
@@ -1,74 +1,29 @@
 journal-header {
-  ion-col {
-    div span {
-      &:first-child {
-        font-size: 2em;
-        font-weight: 900;
-      }
-      font-size: 1.5em;
-      display: block;
-    }
-    &.yesterday,
-    &.tomorrow {
+  span.journal-header-day-navigation {
+    text-decoration: underline;
+    font-size: 16px;
+  }
+  span.journal-header-today-title {
+    font-size: 48px;
+    font-weight: bold;
+  }
+  span.journal-header-date {
+    font-size: 24px;
+    font-weight: bold;
```

Comment:
Could you not use mes-header-lg instead?

Reply:
Yes, I will refactor this now. Perhaps I could also migrate `span.journal-header-today-title` into `_typography.scss` as `mes-title` - thoughts?

Reply:
Can't see another instance of a title like on the journal, will leave that one as it is.